doc/user-guide/logging.md

```diff
@@ -0,0 +1,154 @@
+<!-- start header -->
```

Comment:
So why are we changing all of these guides into markdown?

Reply:
Just changed it to ignore .md files in subdirectories in the doc directory. So we aren't committing all the guides now